src/ray/raylet/worker.cc

```diff
@@ -113,6 +113,14 @@ void Worker::AcquireTaskCpuResources(const ResourceIdSet &cpu_resources) {
   task_resource_ids_.Release(cpu_resources);
 }
 
+const std::unordered_set<ObjectID> &Worker::GetActiveObjectIds() const {
+  return active_object_ids_;
+}
+
+void Worker::SetActiveObjectIds(std::unordered_set<ObjectID> &object_ids) {
+  active_object_ids_ = object_ids;
```

Comment:
You could save a copy here by changing it to `std::unordered_set<ObjectID>` or `std::unordered_set<ObjectID> &&`, and change this line to:
`active_object_ids_ = std::move(object_ids);`

The caller can be changed to use `worker. SetActiveObjectIds(std::move(object_ids));`

Reply:
Done.

Reply:
this needs to be a non-const && to be able to avoid the copy?

also an explicit std::move() should be needed too.